backend/src/bundles/auth/auth.service.ts

```diff
@@ -1,21 +1,60 @@
 import {
+    type UserSignInRequestDto,
+    type UserSignInResponseDto,
     type UserSignUpRequestDto,
     type UserSignUpResponseDto,
 } from '~/bundles/users/types/types.js';
+import { type UserRepository } from '~/bundles/users/user.repository.js';
 import { type UserService } from '~/bundles/users/user.service.js';
 
 class AuthService {
     private userService: UserService;
+    private userRepository: UserRepository;
 
-    public constructor(userService: UserService) {
+    public constructor(
+        userService: UserService,
+        userRepository: UserRepository,
+    ) {
         this.userService = userService;
+        this.userRepository = userRepository;
     }
 
     public signUp(
         userRequestDto: UserSignUpRequestDto,
     ): Promise<UserSignUpResponseDto> {
         return this.userService.create(userRequestDto);
     }
+
+    public async verifyLoginCredentials({
+        email,
+        password,
+    }: UserSignInRequestDto): Promise<{ id: number; email: string }> {
+        const user = await this.userRepository.findByEmail(email);
+
+        if (!user) {
+            throw new Error('Incorrect email');
+        }
+
+        // const isEqualPassword = true await cryptCompare(password, password);
+        // if (!isEqualPassword) {
+        //     throw new Error('Password not match');
+        // }
```

Comment:
This is TODO flow, or you forgot to delete this lines ?

Reply:
I left this code to show that this part is not finished, becuse I need to use cryptCompare or some analogue that was not implemented at that time. Because currently it`s just check if password='HASH', as it was hardcoded inside sign-up route

Reply:
You can try to merge my branch #86, but my branch also uses JWT and this branch too, so it may take a long time to compare and fix.

cryptCompare already included in /backend/src/common/helpers